docs/site/js/getting-started-private.js

```diff
@@ -42,10 +42,11 @@ $(document).ready(function () {
       }
     }
 
-    if (noProxyAddressList && noProxyAddressList.length > 0) {
+    if ((noProxyAddressList && noProxyAddressList.length > 0) || !(noProxyAddressList && noProxyAddressList.length > 0)) {
```

Comment:
```suggestion
    if ((noProxyAddressList && noProxyAddressList.length > 0) || !(noProxyAddressList && noProxyAddressList.length > 0)) {
```
Can you clarify the idea?  Seems weird - `if  x || !x`.  I think the condition don't need changes. 
@trofimovdals